f8a_worker/workers/recommender.py

```diff
@@ -702,8 +736,15 @@ def execute(self, parguments=None):
                                  .format(parguments.get('external_request_id', ''),
                                          set(companion_packages).difference(set(filtered_list))))
 
+                    # Apply License Filters
```

Comment:
This comment is redundant here since you call method `apply_license_filter` which sort of implies that you are going to filter.

Reply:
Yes that is right. But I wanted to highlight that next set of lines of code is related to license filters.